Dockerfile

```diff
@@ -1,48 +1,45 @@
 # ==============================
-FROM helsinkitest/python:3.9-slim as appbase
+FROM registry.access.redhat.com/ubi8/python-39 as appbase
 # ==============================
+
+USER 0
+WORKDIR /app
+
 RUN mkdir /entrypoint
 
-COPY --chown=appuser:appuser requirements.txt /app/requirements.txt
-COPY --chown=appuser:appuser requirements-prod.txt /app/requirements-prod.txt
+COPY --chown=1001:1001 requirements.txt /app/requirements.txt
+COPY --chown=1001:1001 requirements-prod.txt /app/requirements-prod.txt
```

Comment:
How about instead of using magic numbers using the name of the user?

```bash
docker compose up --build
docker exec -it kukkuu-backend id
```
->
```
uid=1001(default) gid=0(root) groups=0(root)
```
i.e. `default`

Reply:
this is changed
`--chown=default:root`